Document/0x06b-Basic-Security-Testing.md

```diff
@@ -334,33 +334,90 @@ Save the IPA file locally with the following command:
 
 #### Installing Apps
 
-When installing apps that are not available via the official distribution channel through Apple's App Store, this is called sideloading. There are various ways of sideloading which are described below.
+When installing apps by not using the official distribution channel through Apple's App Store, this is called sideloading. There are various ways of sideloading which are described below. On the iOS device, the actual installation process is then handled by the installd daemon, which will unpack and install the application. To integrate app services or be installed on an iOS device, all applications must be signed with a certificate issued by Apple. This means that the application can be installed only after successful code signature verification. On a jailbroken phone, however, you can circumvent this security feature with [AppSync](http://repo.hackyouriphone.org/appsyncunified), a package available in the Cydia store. It contains numerous useful applications that leverage jailbreak-provided root privileges to execute advanced functionality. AppSync is a tweak that patches installd, allowing the installation of fake-signed IPA packages.
```

Comment:
> When installing apps by not using the official distribution channel through Apple's App Store,

to

> When you install an application without using Apple's App Store ?


+ Add the ALT text for the url

Reply:
done